documentation/manual-lobster_cpptest.md

```diff
@@ -0,0 +1,115 @@
+# Tracing to C/C++ test requirements
+
+## Setup and requirements
+
+You will need a C/C++ file or a directory containing.
+
+C/C++ files should have one of these extensions to be evaluated by this tool: `.cpp, .cc, .c or .h`
+
+## Preparing C/C++ test comments with requirements
+The test functions should be specified with one the following names: 
+        `TEST`,
+        `TEST_P`,
+        `TEST_F`,
+        `TYPED_TEST`,
+        `TYPED_TEST_P`,
+        `TYPED_TEST_SUITE`,
+        `TEST_P_INSTANCE`,
+        `TEST_F_INSTANCE`
+
+In your test you need to add comments. For example:
+
+```C++
+/**
+ * @requirement CB-#0815, CB-#0816,
+ * 				CB-#0817
+ * @requirement CB-#0818 CB-#0819
+ * 				CB-#0820
+ */
+TEST(ImplicationTest, BasicTest) {}
+```
+
+* Each test can have multiple test methods defined in the comment part.
+
+* These can be simply defined with `/** Test methods */` or `/// Test methods` format. 
+
+* Test methods can be separated by commas or spaces.
```

Comment:
That's true, but as stated above, does the current version of `lobster-cpptest` **do** anything with these test types? Is it possible to test this feature?
If not, then please remove it from the readme file.
Is it a problem if there is a typo in the test methods? If so, then the feature is testable (although useless), and should be mentioned in the readme.

Possibly the next release will provide more functionality related to test methods.

Reply:
I think I used the wrong name for the "Test methods." I meant "Test tags" like "CB—# 0815, CB—#0816, ..."